### PR TITLE
[WIP] #2 Enhance twist control turtle bot

### PR DESCRIPTION
**Pull request by LuisFilipeDias**
Changes:
* Enhance the project a bit, to look more like a project; (/)
* Split application and business logic; (/)
* Create placeholder for tests; (/)
* Say hello to the world; (/)
* Control the TurtleBot from the TWIST node (!) -> In place, but not moving [BUG]
* Setup a launch script, to boot the TWIST, TurtleBot, and any other friends (/)

### CODE REVIEW

**Review by DidFavas — APPROVED** [review]

File: launch/twist_turtlesim.launch [hunk 1]
```diff
@@ -0,0 +1,4 @@
+<launch>
+    <node pkg="turtlesim" type="turtlesim_node" required="true" output="screen" name="turtlesim"></node>
+    <node pkg="twist" type="application" required="true" output="screen" name="twist"></node>
+</launch>
```
Comment (nmssilva) [inline launch/twist_turtlesim.launch]: newline

File: src/business/CMakeLists.txt [hunk 2]
```diff
@@ -0,0 +1,22 @@
+project(Business)
```
Comment (nmssilva) [inline src/business/CMakeLists.txt:1]: small B here

**Review by nmssilva — CHANGES_REQUESTED** [review]
Good start! Just a few small changes.

File: launch/twist_turtlesim.launch (hunk 1 above)
Comment (LuisFilipeDias) [reply launch/twist_turtlesim.launch]: done

File: src/business/CMakeLists.txt (hunk 2 above)
Comment (LuisFilipeDias) [reply src/business/CMakeLists.txt:1]: think its better to change application to upper case. So we have TWIST::Business and not TWIST::business

**Pull request closed without merging**